src/email.c

```diff
@@ -373,14 +373,13 @@ static void *open_connection(void __attribute__((unused)) * arg) {
     pthread_exit((void *)1);
   }
 
-  struct sockaddr_un addr = {
-    .sun_family = AF_UNIX
-  };
+  struct sockaddr_un addr = {.sun_family = AF_UNIX};
```

Comment:
I would undo this change -- it reads better as multiple lines. But up to @octo.

Reply:
The last commit is just changes from running clang-format, as per the contributions guidelines [1], so I'm guessing this is correct but could be missing something.

[1] https://github.com/collectd/collectd/blob/66fd3303352c526255ab4bde256361244e26b259/CONTRIBUTING.md

Reply:
Yeah, this is clang-format, so that's fine.